meal-planner/src/App.js

```diff
@@ -1,59 +1,74 @@
 import { useState } from 'react'
 import styled from 'styled-components/macro'
 import woodBackground from './assets/wood_background.jpg'
-import logo from './assets/logo.png'
+import logo from './assets/logo.svg'
 import Button from './components/Button'
-import MealSelect from './components/MealSelect'
+import MealSelectMenu from './components/MealSelectMenu'
+import { v4 as uuidv4 } from 'uuid'
 
 export default function App() {
-    const weekdays = [
-        'Montag',
-        'Dienstag',
-        'Mittwoch',
-        'Donnerstag',
-        'Freitag',
-        'Samstag',
-        'Sonntag',
-    ]
+    const [weekdays, setWeekdays] = useState([
+        { weekday: 'Montag', id: uuidv4(), meal: '' },
+        { weekday: 'Dienstag', id: uuidv4(), meal: '' },
+        { weekday: 'Mittwoch', id: uuidv4(), meal: '' },
+        { weekday: 'Donnerstag', id: uuidv4(), meal: '' },
+        { weekday: 'Freitag', id: uuidv4(), meal: '' },
+        { weekday: 'Samstag', id: uuidv4(), meal: '' },
+        { weekday: 'Sonntag', id: uuidv4(), meal: '' },
+    ])
 
-    const [isWeekday, setIsWeekday] = useState()
+    const [selectedWeekday, setSelectedWeekday] = useState(null)
 
     function handleClick(event) {
-        setIsWeekday(event.currentTarget.id)
+        setSelectedWeekday(event.currentTarget.id)
+    }
+
+    function selectMeal(meal, id) {
+        setSelectedWeekday(null)
+        const updatedWeekdays = weekdays.map((weekday) =>
+            weekday.id === id ? { ...weekday, meal } : weekday
```

Comment:
Is this working when the user selects more than one meal per day (if intended by you)? 
Maybe ``meal``as an extra object or array helps, so meals can be added instead of replaced.

Reply:
Good point.